pkg/block/adapter.go

```diff
@@ -126,6 +126,8 @@ type CreateMultiPartUploadOpts struct {
 // actually reported.
 type Properties struct {
 	StorageClass *string
+	SizeBytes    *int64
+	Checksum     *string
```

Comment:
StorageClass is pointer as it is optional - by if we set size and checksum as additional properties, I think that all underlying storage should return these values. 
Is it the case or we keep them optional because there are other uses for this struct?

Reply:
I can change `SizeBytes` to `int64` since it's not optional. `Checksum` isn't available for all block adapters.